### PR TITLE
OPS: android scoped storage usage (closes #3089)

### LINKED ISSUE #3089: Change android:requestLegacyExternalStorage="true" to false for enforced scoped storage usage
From AndroidManifest:
https://github.com/BlueWallet/BlueWallet/blob/1bdd37d0fb3d79880fefe5a624b6598572f5a1ca/android/app/src/main/AndroidManifest.xml#L18

change the setting to false in order to use scoped storage that limit not needed access of the app to user data.
also the below change is needed (the addition of `android:maxSdkVersion="28"` to `WRITE_EXTERNAL_STORAGE` permission)
https://github.com/BlueWallet/BlueWallet/blob/1bdd37d0fb3d79880fefe5a624b6598572f5a1ca/android/app/src/main/AndroidManifest.xml#L8

Read for more info: https://developer.android.com/training/data-storage/shared/media#request-permissions

### PR DESCRIPTION
**Pull request by Overtorment**

### CODE REVIEW

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/bg474zndd06du63n96ck7ax2qm?device=pixel4

[download apk](http://filestorage.bluewallet.io:1488/fix-3089-1620318040.apk)

File: android/app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -15,7 +16,7 @@
       android:roundIcon="@mipmap/ic_launcher_round"
       android:allowBackup="false"
       android:extractNativeLibs="true"
-      android:requestLegacyExternalStorage="true"
+      android:requestLegacyExternalStorage="false"
```
Comment (marcosrdz) [inline android/app/src/main/AndroidManifest.xml:19]: I remember I added this line due to a bug where some Android OS version was unable to save PSBTs on their respective Downloads  folder.

Comment (marcosrdz) [reply android/app/src/main/AndroidManifest.xml:19]: Checked out the documentation. Seems like its fine to proceed.

**Review by marcosrdz — APPROVED** [review]

**Comment by marcosrdz** [comment]
@limpbrains  need your review when you get the chance.

**Review by limpbrains — APPROVED** [review]
Sorry, I have no clue what is going on here. 
But it looks OK to me

**Comment by GladosBlueWallet** [comment]
Unbelievable. You, [subject name here], must be the pride of [subject hometown here]!

**Pull request merged**